File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int length = s.length()/2; // 4글자면 2

        if (s.length()%2==0){
            answer+=s.charAt(length-1);
            answer+=s.charAt(length);
        } else {
            answer+=s.charAt(length);
        }
        return answer;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.swift
func solution(_ s:String) -> String {
    var answer = ""
    var temp = Array(s)
    if temp.count % 2 == 0 {
        answer.append(temp[temp.count/2-1])
        answer.append(temp[temp.count/2])
    } else { 
        answer.append(temp[temp.count/2])
    }
    
    return answer
}